microsoftgraph/microsoft-graph-explorer-v2
# Task/handl fetch errors

## Overview

Shows Query Runner errors to customers

### Demo

![error-handling](https://user-images.githubusercontent.com/12011447/57817066-6c6e4600-7786-11e9-81b2-4dc3811d8c8a.JPG)


### Notes

N/A

## Testing Instructions

* Launch Graph Explorer
* Use an invalid query url
* Click `Run Query`
* Observe a red message bar
